pkg/controller/cluster/main_controller.go

```diff
@@ -445,8 +460,8 @@ func (c *Controller) syncHandler(key string) error {
 	// version does not equal the current desired version in the StatefulSet, we
 	// should update the StatefulSet resource.
 	if mi.Spec.Image != ss.Spec.Template.Spec.Containers[0].Image {
-		glog.V(4).Infof("Updating MinIOInstance %s MinIO server version %s, to: %s", name, mi.Spec.Image, ss.Spec.Template.Spec.Containers[0].Image)
-		ss = statefulsets.NewForCluster(mi, svc.Name)
+		klog.V(4).Infof("Updating MinIOInstance %s MinIO server version %s, to: %s", name, mi.Spec.Image, ss.Spec.Template.Spec.Containers[0].Image)
+		ss = statefulsets.NewForCluster(mi, hlSvc.Name)
 		_, err = c.kubeClientSet.AppsV1().StatefulSets(mi.Namespace).Update(ctx, ss, uOpts)
```

Comment:
handle error

Reply:
I just replaced `glog` with `klog` that can be an enhancement on a subsequent PR